gym/envs/toy_text/blackjack.py

```diff
@@ -48,8 +48,8 @@ class BlackjackEnv(gym.Env):
     Face cards (Jack, Queen, King) have point value 10.
     Aces can either count as 11 or 1, and it's called 'usable' at 11.
     This game is placed with an infinite deck (or with replacement).
-    The game starts with each (player and dealer) having one face up and one
-    face down card.
+    The game starts with dealer having one face up and one face down card, while
+    player having two face up card. (Virtually for all Blackjack games today).
```

Comment:
typo `card` -> `cards`?

Reply:
yeah that's just a typo, sorry for my mistake.

Reply:
> typo `card` -> `cards`?

yeah that's just a typo, sorry for my mistake.